conda/recipes/librmm/recipe.yaml

```diff
@@ -72,40 +74,51 @@ outputs:
       ignore_run_exports:
         by_name:
           - cuda-version
+          - if: cuda_major == "11"
+            then: cudatoolkit
     tests:
       - script:
           - "test -d \"${PREFIX}/include/rmm\""
     about:
       homepage: ${{ load_from_file("python/librmm/pyproject.toml").project.urls.Homepage }}
       license: ${{ load_from_file("python/librmm/pyproject.toml").project.license.text }}
       summary: ${{ load_from_file("python/librmm/pyproject.toml").project.description }}
+
   - package:
       name: librmm-tests
       version: ${{ version }}
     build:
       string: cuda${{ cuda_major }}_${{ date_string }}_${{ head_rev }}
-      script:
-        - cmake --install build --component testing
+      script: cmake --install build --component testing
+      dynamic_linking:
+        overlinking_behavior: "error"
     requirements:
       build:
+        - ${{ compiler("c") }}
+        - ${{ compiler("cxx") }}
+        - ${{ compiler("cuda") }}
```

Comment:
These packages are not used here, and their run-exports should be coming through the cache build. This "build step" is _just_ installing (`cmake --install`), it does no compilation. (The example PR in #1800 does actual compilation, so it's different.)

Why do we need them listed here?

Reply:
Everything else in this PR looks right to me. If you can remove the compilers here, I'll approve and merge.

Reply:
Well... now we fail with

```
[bin/gtests/librmm/SYSTEM_MR_TEST] links against:
 └─ libc.so.6
```